taker-frontend/src/App.tsx

```diff
@@ -107,11 +123,14 @@ export const App = () => {
                                     leverage={order?.leverage}
                                     liquidationPrice={liquidationPrice}
                                     walletBalance={walletInfo ? walletInfo.balance : 0}
+                                    opening_fee={openingFee}
+                                    funding_fee_annualized_percent={fundingRateAnnualized}
                                 />
                                 <History
                                     connectedToMaker={connectedToMaker}
                                     cfds={cfds.filter((cfd) => cfd.state.getGroup() !== StateGroupKey.CLOSED)}
                                     title={"Open Positions"}
+                                    isClosed={false}
```

Comment:
We split the CFDs into two separate lists and I wanted to adhere to that. This just keeps the logic of "splitting it" in one place, otherwise I have to reason twice...

Reply:
I changed it into something more specific

Reply:
> We split the CFDs into two separate lists and I wanted to adhere to that. This just keeps the logic of "splitting it" in one place, otherwise I have to reason twice - one time outside of `Histroy` and one time inside...

You always reason about the same thing though which is the state of the CFD? You can easily keep this DRY by extracting that in a function.

What we currently have is pretty confusing because in order to understand what happens, you need to know that the reason this is hard-coded to `false` here is because _a different_ piece of code groups these CFDs into closed and non-closed.

So now you have semantic coupling that is not enforced by the code. If the condition on how you split these CFDs changes without updating this flag, the code inside the tiles will behave incorrectly. The CFD tile already knows its state, it should reason about that locally and not require it to be told externally something that it already knows.

Reply:
extracted function `isClosed(cfd: Cfd): boolean` - tried to have the `Cfd` as `class` first so I can have a method, but could not easily achieve the mapping using the class, so I opted for a function that reasons about the `Cfd` instead.